src/App.js

```diff
@@ -21,31 +21,34 @@ function App() {
   });
 
   useEffect(() => {
-    const userId = getLoggedInUserId();
-    if (userId === null) {
-      return;
-    }
-    client
-      .get(`/user/${userId}`)
-      .then(res => setUser(res.data.data.user))
-      .catch(err => console.log(err));
+    const userId = getLoggedInUserId()
+    if (userId === null) { return }
```

Comment:
What's the reason behind putting this on one line? It breaks the consistency of the rest of the teams

Reply:
There's no reason other than using less lines of code. 
I'll change it to suit everyone else.